src/app/components/drawer/base-drawer.tsx

```diff
@@ -61,70 +62,52 @@ export const BaseDrawer = memo((props: BaseDrawerProps) => {
   const onGoBack = () => navigate(-1);
 
   return (
-    <Flex
-      display={isShowing ? 'flex' : 'none'}
-      bg={`rgba(0,0,0,0.${isShowing ? 4 : 0})`}
-      transition={transition}
-      position="fixed"
-      top={0}
-      left={0}
-      height="100%"
-      pt="loose"
-      width="100%"
-      alignItems={['flex-end', 'center', 'center']}
-      justifyContent="center"
-      flexDirection="column"
-      zIndex={1000}
-      style={{
-        pointerEvents: !isShowing ? 'none' : 'unset',
-        userSelect: !isShowing ? 'none' : 'unset',
-        willChange: 'background',
-      }}
-      {...rest}
-    >
-      <Flex
-        flexDirection="column"
-        flexGrow={0}
-        ref={ref}
-        opacity={isShowing ? 1 : 0}
-        transform={isShowing ? 'none' : 'translateY(35px)'}
-        transition={isShowing ? transition + ' 0.1s' : transition}
-        transitionDuration="0.4s"
-        willChange="transform, opacity"
-        width="100%"
-        maxWidth="472px"
-        bg={theme === 'dark' ? token('colors.black') : token('colors.white')}
-        borderTopLeftRadius="16px"
-        borderTopRightRadius="16px"
-        borderBottomLeftRadius={[0, '16px', '16px', '16px']}
-        borderBottomRightRadius={[0, '16px', '16px', '16px']}
-        position="relative"
-        mt={['auto', 'unset', 'unset', 'unset']}
-        maxHeight={['calc(100vh - 24px)', 'calc(100vh - 96px)']}
-      >
-        <Box
-          css={{
-            overflowY: 'scroll',
-
-            '&::-webkit-scrollbar': {
-              display: 'none',
-            },
-          }}
+    <Dialog.Root open={isShowing} onOpenChange={onClose}>
```

Comment:
I think you'll need to specify the BG colour for this dialog, otherwise it will be transparent in dark mode. We probably need to make a wrapper for `Dialog` 

I had to fix this in a couple of places, [see here](https://github.com/leather-wallet/extension/pull/4283)

Reply:
there is flex wrapper with bg color inside, but yeah, creation of wrapper is a good idea. but not in this pr